cookbooks/cdo-varnish/libraries/helpers.rb

```diff
@@ -196,7 +196,7 @@ def process_proxy(behavior, app)
 
 # Returns the hostname-specific conditional expression for the app provided.
 def if_app(app, req)
-  app == :dashboard ? (req + '.http.host ~ "(dashboard|studio).code.org$"') : nil
+  app == :dashboard ? (req + '.http.host ~ "(dashboard|studio)"') : nil
```

Comment:
Because wildcard certificates only cover a single subdomain level (for a `*.example.com` certificate `sub.example.com` is ok but `sub.sub.example.com` is not), the naming convention we've currently adopted uses either `dashboard-subdomain.example.com` or `subdomain-studio.example.com` to recognize a Code Studio hostname. This is as restricted as this can be for now.

I've started looking into replacing this messy regex-match entirely by providing the dashboard/pegasus hostnames being used to the cdo-varnish cookbook directly via attributes, then we should have full control over the hostnames used by each webapp moving forward. I'll need to do a bit more testing to make sure it doesn't adversly affect any of our live-site deployments though.